src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 0 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__list:last-child {
+  margin-right: 50px;
+}
```

Comment:
```suggestion
.nav__list {
  margin-right: 50px;
}
```

Reply:
You can just add this margin to class, no need to increase selector specifity

Reply:
oh.. really)
ty so much!)